Code/Tools/TestImpactFramework/Runtime/Common/Code/Include/Static/BuildTarget/Common/TestImpactBuildTargetList.h

```diff
@@ -47,37 +49,52 @@ namespace TestImpact
         //! @param outputName The output name of the target to get.
         AZStd::string GetTargetNameFromOutputNameOrThrow(const AZStd::string& outputName) const;
 
+        //! Get the vector of build targets in the repository.
```

Comment:
uber nit: Could omit `vector` here and just say 'Get the build targets in the repository` (so we don't encode type information in the comment in case it changes in future)

Reply:
Fixed 👍